sql/server/src/main/java/io/stargate/api/sql/server/avatica/StargateMeta.java

```diff
@@ -34,31 +36,33 @@
 
   private final ConcurrentMap<String, Connection> connections = new ConcurrentHashMap<>();
 
-  private final DataStore dataStore;
+  private final DataStoreFactory dataStoreFactory;
   private final AuthenticationService authenticator;
 
-  public StargateMeta(DataStore dataStore, AuthenticationService authenticator) {
-    this.dataStore = dataStore;
+  public StargateMeta(DataStoreFactory dataStoreFactory, AuthenticationService authenticator) {
+    this.dataStoreFactory = dataStoreFactory;
     this.authenticator = authenticator;
   }
 
   private void newConnection(ConnectionHandle ch, Map<String, String> info) {
     connections.computeIfAbsent(
         ch.id,
         sid -> {
-          String username = info.get("user");
+          String username;
           String password = info.get("password");
```

Comment:
Could the be called "token" as to differentiate it from user/pass authentication in the future?

Reply:
Ah, nm. I see how it's being passed.

Reply:
The name "password" is kind of fixed by JDBC, IIRC. We could require "token" to be the user name for clarity, WDYT?

Reply:
This is what CQL does. It uses a configurable setting for the token user's name, but it defaults to "token".

Reply:
added code to enforce the "token" user name in JDBC connections.